File: Server/Models/Song.cs
namespace Server.Models;

public class Song
{
    public Guid Id { get; set; }
    public String Title { get; set; } = string.Empty;
    public String Artist { get; set; } = string.Empty; 
    public int ReleaseYear { get; set; } 
    public String SpotifyId { get; set; } = string.Empty;
    public String PreviewUrl { get; set; } = string.Empty; 
    public String Language { get; set; } = "English";
}

File: Client/src/types/song.ts
// Song Type
export interface Song {
  id: string; // Guid -> string
  title: string;
  artist: string; 
  releaseYear: number; 
  spotifyId: string;
  previewUrl: string; 
  language: string;
}

// TimelineCard Type
export interface TimelineCard {
  id: string; // Guid -> string
  songId: string; // Guid -> string
  title: string;
  artist: string;
  releaseYear: number;
}